jenkins-build-driver/src/test/java/org/jboss/pnc/jenkinsbuilddriver/test/JenkinsDriverRemoteTest.java

```diff
@@ -143,12 +142,12 @@ public RepositorySession getRepositorySession() {
             
             @Override
             public String getJenkinsUrl() {
-                return System.getProperty("PNC_JENKINS_URL");
+                return System.getenv("PNC_JENKINS_URL") + ":" + getJenkinsPort();
```

Comment:
Try to avoid using env properties it requires that user set them in order to run tests. Env variables we are using is just one option to pass values to the configuration. Can you use org.jboss.pnc.common.Configuration ?